### DISCUSSION TITLE
Where do I file, vote on, and find workarounds for feature requests regarding code search and navigation?

### DISCUSSION THREAD

**Discussion by pozega** [community/community#68444] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Where do I file, vote on, and find workarounds for feature requests regarding code search and navigation?

**Comment by look** [comment, marked as answer]
This category ([Code Search and Navigation](https://github.com/orgs/community/discussions/categories/code-search-and-navigation)) is the right place for feedback about code search and code navigation.

**Comment by utkarshsingx** [comment]
On GitHub, you can file, vote on, and find workarounds for feature requests related to code search and navigation in the **GitHub Issues** section of the respective repository for the GitHub product you are using. Here's how you can do it:

1. **Filing a Feature Request:**
    - Go to the repository of the GitHub product you are using.
    - Click on the **"Issues"** tab at the top of the repository.
    - Click the green **"New Issue"** button.
    - Choose the **"Feature request"** template.
    - Describe your feature request in detail and submit the issue.

2. **Voting on Feature Requests:**
    - Visit the **"Issues"** section of the repository.
    - Find the feature request you want to support.
    - Click the **"Thumbs up"** reaction (👍) to upvote the request. The more reactions a request gets, the more attention it receives.

3. **Finding Workarounds:**
    - Workarounds are often discussed in the comments of related issues.
    - Read through the comments on the issue to see if there are any community-suggested workarounds or temporary solutions.

**Reply by setharnold** [reply]
Can you suggest how to find the appropriate GitHub repository for any specific UI element? At the moment, https://github.com/orgs/github/repositories has 503 repositories. I want to suggest a feature for the `Issues` tab, but there's no repository named "GitHub" https://github.com/github/github --> `404`.

None of these repositories look quite right to me: https://github.com/orgs/github/repositories?q=issues

Thanks

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐

**Comment by lgroh** [comment]
Is there a way to get to the first commit of a repo without clicking 'older' until you get there? If not, I think this would be a really helpful feature to add. Maybe a first commit button somewhere?

Example: We have quite a few legacy projects. One of our clients wanted something done to a legacy project and our project manager needed to share the first commit with the client to make sure we were on the same page about how old this project really is. I clicked through dozens of pages to get to the first commit.

Thank you for your time!

**Reply by Ron7310** [reply]
From the command line, inside a more display?  On my computer you can type the End key and it will bring you to the end.

You can also just run "git log --reverse"...
